proto/cosmos/bank/v1beta1/cli.yaml

```diff
@@ -0,0 +1,22 @@
+tx:
+  service: cosmos.bank.v1beta1.Msg
+  method_options:
+    Send:
+      use: send [from_key_or_address] [to_address] [amount...]
+      field_options:
```

Comment:
why is this called field_options? Should be args or argument. Also, how do we handle flags?

Reply:
This is all based on protobuf services, methods, fields, etc. By default all fields become flags. This let's us customize that. I'll try to be more descriptive in the doc strings